src/main/java/core/basesyntax/services/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.services;
+
+import core.basesyntax.figures.*;
+
+import java.util.Random;
+
+import static core.basesyntax.services.Color.WHITE;
+
+public class FigureSupplier {
+
+    private static final int RANGE_OF_RANDOM_SIZE = 100;
+    private static final int RADIUS_FOR_DEFAULT_FIGURE = 10;
+
+    public Figure getRandomFigure() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
New instance of ```colorSupplier``` will be created during ```getRandomFigure()``` call. Isn't it better to make ```colorSupplier``` a class-level property?

Reply:
Thank you! Done.